web/profiles/custom/os2loop/modules/os2loop_share_with_a_friend/src/Form/ShareWithAFriendForm.php

```diff
@@ -152,11 +147,15 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $send = TRUE;
     $result = $this->mailManager->mail($module, $key, $to, $langcode, $params, NULL, $send);
     if ($result['result'] !== TRUE) {
-      $this->messenger->addMessage($this->t('There was a problem sending your message and it was not sent.'), 'error');
+      $this->messenger->addError($this->t('There was a problem sending your message and it was not sent.'), 'error');
```

Comment:
Remove `, 'error'`. And if you ever need to pass the message type, use one of the constants in https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Messenger%21MessengerInterface.php/interface/MessengerInterface/9.2.x.